### ISSUE TITLE
Running on physical device

### ISSUE THREAD

**Issue by willsam100** [fsprojects/Canopy.Mobile#31]
Can this be run against a physical device? 
I see AVD in the configuration, suggesting it only runs with simulators for now.

**Comment by forki** [comment]
Yeah I guess that would work if the avd settings are changed. We would accept PRs for that
